utils/docker-ce/files/dockerd.init

```diff
@@ -3,12 +3,101 @@
 USE_PROCD=1
 START=25
 
+DOCKERD_CONF="/tmp/dockerd/daemon.json"
+
+json_add_array_string() {
+	json_add_string "" "$1"
+}
+
+process_config() {
+	local data_root log_level
+
+	[ -f /etc/config/dockerd ] || return 0
+
+	config_load 'dockerd'
+	config_get data_root globals data_root "/opt/docker/"
+	config_get log_level globals log_level "warn"
+
+	. /usr/share/libubox/jshn.sh
+	json_init
+	json_add_string "data-root" "$data_root"
+	json_add_string "log-level" "$log_level"
+	json_add_array "registry-mirrors"
+	config_list_foreach globals registry_mirror json_add_array_string
+	json_close_array
+
+	mkdir -p /tmp/dockerd
+	json_dump > "$DOCKERD_CONF"
+}
+
 start_service() {
 	local nofile=$(cat /proc/sys/fs/nr_open)
 
+	process_config
+
 	procd_open_instance
 	procd_set_param stderr 1
-	procd_set_param command /usr/bin/dockerd
+	procd_set_param command /usr/bin/dockerd --config-file="$DOCKERD_CONF"
 	procd_set_param limits nofile="${nofile} ${nofile}"
 	procd_close_instance
 }
+
+ip4tables_remove_nat() {
+	iptables -t nat \
```

Comment:
We are using a fair amount of vertical space for these commands, IMO we could just have one command per line.
Also, are these all the commands that get added?